.changeset/beige-ants-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+"gradio": patch
```

Comment:
Should be a minor since we're updating the API. 

Can you also add a highlight, [like done in this PR](https://github.com/gradio-app/gradio/pull/5057), showing how to use the new API to update the components?